vespa-feed-client-cli/src/main/java/ai/vespa/feed/client/impl/CliArguments.java

```diff
@@ -58,6 +58,7 @@ class CliArguments {
     private static final String STDIN_OPTION = "stdin";
     private static final String DOOM_OPTION = "max-failure-seconds";
     private static final String PROXY_OPTION = "proxy";
+    private static final String GZIP_REQUESTS_OPTION = "gzip-requests";
```

Comment:
I think --compression=gzip is better.
Then it is very easy to support other compression schemes later on.

Reply:
That's a very good suggestion. I'll change it.